common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2145,7 +2145,7 @@ object CelebornConf extends Logging {
       .withAlternative("celeborn.client.maxRetries")
       .internal
       .categories("client", "worker")
-      .doc("Max retry times for client to connect master endpoint")
+      .doc("Max retry times for client to connect master endpoint. The minimum times of max retry times is the size of master endpoints.")
```

Comment:
This seems little confusing. How about this?

```suggestion
      .doc("Max retry times for client to connect master endpoint. The number of retries will be at least equal to the number of master endpoints.")
```